diff --git a/node_modules/react-native-svg/lib/module/web/utils/prepare.js b/node_modules/react-native-svg/lib/module/web/utils/prepare.js
index 796b489..b7e8cb6 100644
--- a/node_modules/react-native-svg/lib/module/web/utils/prepare.js
+++ b/node_modules/react-native-svg/lib/module/web/utils/prepare.js
@@ -1,4 +1,4 @@
-import { hasTouchableProperty, parseTransformProp } from '.';
+import { hasTouchableProperty, parseTransformProp } from './index';
 import { resolve } from '../../lib/resolve';
 import { resolveAssetUri } from '../../lib/resolveAssetUri';
 /**
